src/components/SideNav.tsx

```diff
@@ -13,13 +13,13 @@ import docLinks from "../data/developer-docs-links.yaml"
 
 export const dropdownIconContainerVariant = {
   open: {
-    rotate: 0,
+    rotate: 90,
     y: 3,
```

Comment:
```suggestion
```

The framer positioning can be removed due to the flex container being added.

Reply:
Ah right! nice catch. Lets remove that as well.